app/overrides/add_documents_to_main_navigation.rb

```diff
@@ -1,6 +1,6 @@
 Deface::Override.new(
-  virtual_path:  'spree/layouts/admin',
-  insert_bottom: '#main-sidebar',
+  virtual_path:  'spree/admin/shared/_main_menu',
+  insert_bottom: 'nav',
   partial:       'spree/admin/shared/menu/documents_tab',
   name:          'documents_tab'
 )
```

Comment:
Could you remove this changes from PR.

We should have it in master but with another PR.

Also let's keep support  Spree < 4
```
if Spree.version.to_f < 4.0
  Deface::Override.new(
    virtual_path:  'spree/layouts/admin',
    insert_bottom: '#main-sidebar',
    partial:       'spree/admin/shared/menu/documents_tab',
    name:        'documents_tab'
  )
else
  Deface::Override.new(
    virtual_path:  'spree/admin/shared/_main_menu',
    insert_bottom: 'nav',
    partial:       'spree/admin/shared/menu/documents_tab',
    name:          'documents_tab'
  )
end
```

Reply:
no problem. Let me create another PR.

Reply:
> no problem. Let me create another PR.

Let's add to PR #143

Reply:
Added to #143

Reply:
Could you just revert file with `git checkout master app/overrides/add_documents_to_main_navigation.rb`

Reply:
updated.